docs/docs/recipes/protect-your-api/README.md

```diff
@@ -2,4 +2,269 @@
+  appId: "<your-application-id>"
+  resources: ["<your-api-resource>"]
+)
+let logtoClient = LogtoClient(useConfig: config)
+```
+
+</TabItem>
+
+<TabItem value="react" label="React">
+
+```js
+import { LogtoProvider, LogtoConfig } from '@logto/react';
+
+const config: LogtoConfig = {
+  appId: '<your-application-id>',
+  endpoint: '<your-logto-endpoint>',
+  resources: ['<your-api-resource>'],
+};
+
+const App = () => <LogtoProvider config={config}>{/* Your app content */}</LogtoProvider>;
+```
+
+</TabItem>
+<TabItem value="vue" label="Vue">
+
+```js
+import { createLogto, LogtoConfig } from '@logto/vue';
+
+const config: LogtoConfig = {
+  appId: '<your-application-id>',
+  endpoint: '<your-logto-endpoint>',
+  resources: ['<your-api-resource>'],
+};
+
+const app = createApp(App);
+
+app.use(createLogto, config);
+app.mount('#app');
+```
+
+</TabItem>
+<TabItem value="js" label="VanillaJs">
+
+```js
+// TODO:
+```
+
+</TabItem>
+</Tabs>
+
+Let's take a breath here. Your application is almost ready 😊. Logto SDK is well configured at this point. Users could sign in / sign up through the UI now.
+
+After end-users successfully log in, they may try to get access to some of your backend API resources through the client application upon their request. We'll need to loop Logto in again at this point. Call Logto server through our SDK to request an `access_token` before making the actual API request.
+
+:::note
+
+The requested target resource should be:
+
+1. Registered in Logto server.
+2. Initiated as configs into the Logto SDK at your application.
+
+:::
+
+<Tabs>
+<TabItem value="kotlin" label="Kotlin">
+
+```kotlin
+logtoClient.getAccessToken("<your-target-api-resource>", { logtoException: LogtoException?, result: AccessToken? ->
+  // custom logic
+})
+```
+
+</TabItem>
+<TabItem value="java" label="Java">
+
+```java
+logtoClient.getAccessToken("<your-target-api-resource>", (logtoException, accessToken) -> {
+    // custom logic
+});
+```
+
+</TabItem>
+<TabItem value="swift" label="Swift">
+
+```swift
+  let token = try await client.getAccessToken(for: "<your-target-api-resource>")
+  // custom logic
+```
+
+</TabItem>
+<TabItem value="react" label="React">
+
+```js
+const { getAccessToken } = useLogto();
+const token = await getAccessToken('<your-target-api-resource>');
+// custom logic
+```
+
+</TabItem>
+
+<TabItem value="vue" label="Vue">
+
+```js
+const { getAccessToken } = useLogto();
+const token = await getAccessToken('<your-target-api-resource>');
+// custom logic
+```
+
+</TabItem>
+
+<TabItem value="js" label="VanillaJs">
+
+```js
+const token = await logtoClient.getAccessToken('<your-target-api-resource>');
+// custom logic
+```
+
+</TabItem>
+
+</Tabs>
+
+If the client is well-authorized, a JWT format `access_token` will be granted and issued by Logto, specifically for the requested entity. Encrypted, audience_restricted, and with lift-time. Carrying all the necessary info that can represent the authority of this request.
```

Comment:
Should be `audience-restricted` and `Carry`?
```suggestion
If the client is well-authorized, a JWT format `access_token` will be granted and issued by Logto, specifically for the requested entity. Encrypted, audience-restricted, and with lift-time. Carry all the necessary info that can represent the authority of this request.
```

Reply:
No, I think they are still in one sentence. `encrypted, audience-restricted and with life-time, carrying all the necessary info` are still modifying the preceding noun `access token`

Reply:
Btw, there is a typo `lift-time` -> `life-time`, and `audience_based` should be `audience-based` with hyphen.

Reply:
Maybe this sentence is too long to read. Grammarly may split it.

Reply:
Yep, use the whole sentence as an adj.  Intend to keep the sentence short to read.